frontend/public/locales/es/translation.json

```diff
@@ -0,0 +1,279 @@
+  "Sorry, there was a problem. Try a refresh or navigate to a different page.": "Sorry, there was a problem. Try a refresh or navigate to a different page.",
+  "Subset": "Subset",
+  "Switch language": "Cambiar idioma",
+  "Switch to {{theme}} Mode": "Cambiar a modo {{theme}}",
+  "System Error": "System Error",
+  "TCP Connection": "TCP Connection",
+  "TCP Traffic": "TCP Traffic",
+  "The meshConfig.meshMTLS.minProtocolVersion field specifies the minimum TLS version for the TLS                   connections among Istio workloads. N/A if it was not set.": "The meshConfig.meshMTLS.minProtocolVersion field specifies the minimum TLS version for the TLS                   connections among Istio workloads. N/A if it was not set.",
+  "The Side Panel shows details about the currently selected node or edge, otherwise the whole graph.": "The Side Panel shows details about the currently selected node or edge, otherwise the whole graph.",
+  "The Side Panel shows details about the currently selected node or edge, otherwise the whole mesh.": "The Side Panel shows details about the currently selected node or edge, otherwise the whole mesh.",
+  "This chart shows cpu consumption for the istiod {{cpuMetricSource}}": "This chart shows cpu consumption for the istiod {{cpuMetricSource}}",
+  "This chart shows memory consumption for the istiod {{memoryMetricSource}}": "This chart shows memory consumption for the istiod {{memoryMetricSource}}",
+  "This value represents the delay in seconds between config change and a proxy receiving all required                 configuration.": "This value represents the delay in seconds between config change and a proxy receiving all required                 configuration.",
+  "This value represents the meshConfig.outboundTrafficPolicy.mode, that configures the sidecar handling of                 external services, that is, those services that are not defined in Istio’s internal service registry. If                 this option is set to ALLOW_ANY, the Istio proxy lets calls to unknown services pass through. If the                 option is set to REGISTRY_ONLY, then the Istio proxy blocks any host without an HTTP service or service                 entry defined within the mesh": "This value represents the meshConfig.outboundTrafficPolicy.mode, that configures the sidecar handling of                 external services, that is, those services that are not defined in Istio’s internal service registry. If                 this option is set to ALLOW_ANY, the Istio proxy lets calls to unknown services pass through. If the                 option is set to REGISTRY_ONLY, then the Istio proxy blocks any host without an HTTP service or service                 entry defined within the mesh",
+  "Time duration": "Time duration",
+  "Time range": "Time range",
+  "Time Range": "Time Range",
+  "Time Range & Replay": "Time Range & Replay",
+  "Traffic": "Tráfico",
+  "Traffic Animation": "Traffic Animation",
+  "Traffic Graph": "Tráfico",
```

Comment:
I have translated `Traffic Graph` to just `Tráfico` because `Gráfico del Tráfico` sounds a bit weird. Any proposal is welcome :slightly_smiling_face:

Reply:
Yes, it sound a bit odd :)